src/main/resources/db/migration/all/20240425143644__update_information_request_statuses.sql

```diff
@@ -0,0 +1,20 @@
+UPDATE
+    approved_premises_applications
+SET
+    status = 'REQUESTED_FURTHER_INFORMATION'
+WHERE
+        id IN (
+        SELECT
+            application_id
+        from
+            assessments
+        where
+                id IN (
+                SELECT
+                    assessment_id
+                from
+                    assessment_clarification_notes
+                where
+                    response is null
+            )
+    );
```

Comment:
to help my understanding, i think this query is the equivalent of this:

```
UPDATE
    approved_premises_applications
SET
    status = 'REQUESTED_FURTHER_INFORMATION'
WHERE
        id IN (
              select application_id from assessments assess
              inner join assessment_clarification_notes acn on acn.assessment_id = assess.id
              where acn.response is null
        )
);
```

Reply:
Yeah, that's right 👍